src/main/resources/db/migration/0250/V266__CohortModuleTitle.sql

```diff
@@ -0,0 +1,13 @@
+-- Set new column sequence to number orderings
+BEGIN;
```

Comment:
Flyway automatically runs each migration in a transaction, so no need for begin/commit.